apps/chat/src/components/Files/FileManagerModal.tsx

```diff
@@ -48,6 +49,7 @@ export const FileManagerModal = ({
   allowedTypes = [],
   initialSelectedFilesIds = [],
   isInConversation = false,
+  customHeaderLabel,
```

Comment:
```suggestion
  customHeaderLabel = t('Manage attachments'),
```

Reply:
in this case let's name it as `headerLabel`

Reply:
It is not possible to do `customHeaderLabel = t('Manage attachments'),`
We define **t** below in the component.
`const { t } = useTranslation(Translation.Settings);` 

instead changed this constant definition
```
  const headerLabel = customHeaderLabel
    ? customHeaderLabel
    : t('Manage attachments');
```